ingredient/ingredient-databricks/src/plugin.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseIngredient, IngredientManager } from "@azbake/core"
+import { ARMHelper } from "@azbake/arm-helper"
+import ARMTemplate from "./arm.json"
+export class DataBricks extends BaseIngredient {
+
+    public async Execute(): Promise<void> {
+        try {
+            let util = IngredientManager.getIngredientFunction("coreutils", this._ctx)
+            this._logger.log('Logical SQL Server Plugin Logging: ' + this._ingredient.properties.source)
+
+            const helper = new ARMHelper(this._ctx);
+
+            let params = await helper.BakeParamsToARMParamsAsync(this._name, this._ingredient.properties.parameters)
+            
+            if (!params["diagnosticsEnabled"])
```

Comment:
A simple approach is now available.  See this for an example.
https://github.com/HomecareHomebase/azure-bake/blob/8e4a76c0bddf352c680005cf4c5ac1bc8bf0f60b/ingredient/ingredient-traffic-manager/src/plugin.ts#L46